openvmm/hvlite_helpers/src/disk.rs

```diff
@@ -14,7 +14,7 @@ use vm_resource::Resource;
 /// .vhdx, the file will be opened using the kernel-mode VHD parser.
 pub fn open_disk_type(path: &Path, read_only: bool) -> anyhow::Result<Resource<DiskHandleKind>> {
     Ok(match path.extension().and_then(|s| s.to_str()) {
-        Some("vhd") => {
+        Some("vhd" | "vmgs") => {
```

Comment:
I'm worried about lumping in these two under one match arm, given that "vmgs" is (from what I understand) always a fixed-size VHD. I'd prefer if we encoded the current semantics as accurately as possible for the time being.

Reply:
Yeah fair enough